relayer/src/link/relay_path.rs

```diff
@@ -941,12 +957,21 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> RelayPath<ChainA, ChainB> {
             .send_messages_and_wait_commit(tm)
             .map_err(LinkError::relayer)?;
 
-        info!("result: {}", PrettyEvents(&dst_tx_events));
+        info!(
+            "result: {}",
+            PrettyEvents(
+                dst_tx_events
+                    .iter()
+                    .map(|ev| ev.event.clone())
```

Comment:
Is there no way around cloning all the events here? Feel free to disregard

Reply:
[714b03c](https://github.com/informalsystems/ibc-rs/pull/2542/commits/714b03c7ecd737684e64da95dfe9a76837db72e0)